### PR TITLE
Initial checkin

### PR DESCRIPTION
**Pull request by eduardisimo203**

### CODE REVIEW

File: log/log_examples_test.go [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.comcast.com/dna/go-log/log"
```
Comment (ghost) [inline log/log_examples_test.go]: Change import path

File: log/log_test.go [hunk 2]
```diff
@@ -0,0 +1,990 @@
...
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.comcast.com/dna/go-log/log"
```
Comment (ghost) [inline log/log_test.go]: Change import path

Comment (ghost) [reply log/log_test.go]: fixed

File: log/log_examples_test.go (hunk 1 above)
Comment (ghost) [reply log/log_examples_test.go]: fixed

**Comment by JohnRiv** [comment]
When you're ready to merge this in, please be sure to use the "Squash and merge" option

**Pull request merged**